tests/src/test/scala-2.13+/cats/tests/ArraySeqSuite.scala

```diff
@@ -60,13 +60,13 @@ class ArraySeqSuite extends CatsSuite {
   }
 
   test("show") {
-    ArraySeq(1, 2, 3).show should ===(s"ArraySeq(1, 2, 3)")
-    ArraySeq.empty[Int].show should ===(s"ArraySeq()")
+assert(    ArraySeq(1, 2, 3).show === (s"ArraySeq(1, 2, 3)"))
+assert(    ArraySeq.empty[Int].show === (s"ArraySeq()"))
```

Comment:
o.O

Reply:
Ah damnit, need to run scalafmt for 2.13

Reply:
by the way, you could think of introducing a CatsSuite trait which defines an assertEq method using Eq. This pattern is how we use MUnit in Metals, where we have a few custom assertion methods built on top of the stock ones